### PR TITLE
Create contact section

### PR DESCRIPTION
**Pull request by Ismailco**
- Create the contact section
- Create the footer
- Add contact form
- Link form spree
- Add fontawesom icons

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -307,3 +308,111 @@ body {
...
+  border-radius: 4px;
+  background-color: #fff;
+  border: 1px solid #dfdfe0;
+  margin-left: 18px;
+  padding: 16px;
+}
+
+.contact_btn {
+  margin-top: 20px;
+}
```
Comment (brytebee) [inline style.css]: - [x] **Required:** The contact_btn should visually change when the button is hovered over or/and clicked. This is to visually indicate interaction and add some aesthetics to your website. I hope it makes some sense 😏.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @Ismailco👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- ✔️Appealing UI design
- ✔️ Clean code and proper use of GitHub flow
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️

**README.md**

- [x] **Required:** Add a screenshot of your website
- [ ] **Required:** Complete the Readme documentation by completing information such as Prerequisite, usage, etc. 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -152,5 +152,57 @@ <h2 class="about_headline">About</h2>
...
+        <input
+          class="input2"
+          type="email"
+          name="email"
+          placeholder="exemple@mail.com"
+          required
+        />
+        <textarea
+          class="input3"
+          name="message"
```
Comment (micheaol) [inline index.html:182]: [OPTIONAL]: Please ensure you stick to the design as much as possible (e.g., font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

## Your design:
your-design

## Figma design: 
figma

**Review by micheaol — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 
Hi @Ismailco , 

Great work on making the changes requested by a previous reviewer 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

well-1

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**